docs/application/dotnet/guides/app-management/cion.md

```diff
@@ -0,0 +1,434 @@
+# Cion
+
+The Cion API provides functionality to communicate with the other devices.
+
+The main features of the `Tizen.Applications.Cion` class include:
+
+- Communicating with other applications in server-client style.
+
+  You can communicate with other applications in [server-client](#server_client) style.
+
+- Communicating with other applications in group style.
+
+  You can communicate with other applications in [group](#group) style.
+
+And, you can generate communicate code using [TIDL](#tidl).
+
+## Prerequisites
+
+To enable your application to use the cion functionality, follow these steps:
+
+1.  To use the [Tizen.Applications.Cion](https://docs.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.Cion.html) class, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
```

Comment:
@mijin85cho , Even though the link has been updated, the link is still broken. I am assuming issue with the symbolic link "latest" has not been fixed yet. Could you please check and help to fix the link.

Reply:
link is changed to API9 now

Reply:
changed to latest